metainsuranceorg.py

```diff
@@ -407,7 +414,9 @@ def process_newrisks_insurer(self, risks_per_categ, number_risks_categ, acceptab
                                                            expire_immediately=self.simulation_parameters[
                                                                "expire_immediately"], )
                             self.underwritten_contracts.append(contract)
+                            self.cash_left_by_categ = cash_left_by_categ
                             risks_per_categ[categ_id][iter] = None
+                            self.cash_left_by_categ = cash_left_by_categ
```

Comment:
Why is this line duplicated?

Reply:
We need to update self.cash_left_by_categ whenever a new insurance or reinsurance contract is underwritten. So we need it for both cases.

Reply:
Now, yes.

Done!!!!